### ISSUE TITLE
LOG_LEVEL has no effect

### ISSUE THREAD

**Issue by webJose** [medikoo/log-node#7]
Hello.  I have tried various levels and nothing seems to work.  It always outputs debug lines and up.

**Comment by medikoo** [comment]
@webJose What output do you see, if you run following script with Node.js?

```javascript
process.env.LOG_LEVEL = 'info';

require('log-node')();

const log = require('log');

log.debug('Hidden');
log.info('Visible');
```

**Comment by webJose** [comment]
Hello @medikoo , thank you for stopping by.  Your test works as expected my problem was that I could not make it work as I needed in the big project.

I just opened a file and saw this:

```js
const { port, environment, title, version, logLevel, logDebug } = require('../config/app-settings');
require('log-node')({
  env: {
    LOG_LEVEL: logLevel,
    LOG_DEBUG: logDebug
  }
});
```

It seems someone found out that `log-node` can be set up like this?  I did not know.  I'll test by changing the source of app-settings, but this is most likely mi issue.  I'll probably have to apologize for not noticing this. :-)
